src/test/system_tests/test_utils.js

```diff
@@ -422,6 +425,67 @@ function get_new_buckets_path_by_test_env(new_buckets_full_path, new_buckets_dir
     return is_nc_coretest ? path.join(new_buckets_full_path, new_buckets_dir) : new_buckets_dir;
 }
 
+// /**
+//  * create_fresh_config_sub_dirs_manual deletes and creates config dir subdirectories manualy 
+//  * @param {string} config_root 
+//  * @returns {Promise<Void>}
+//  */
+// async function create_fresh_config_sub_dirs_manual(config_root) {
+//     await P.all(_.map([CONFIG_SUBDIRS.IDENTITIES, CONFIG_SUBDIRS.ACCOUNTS_BY_NAME,
+//         CONFIG_SUBDIRS.BUCKETS, CONFIG_SUBDIRS.ACCESS_KEYS], async dir =>
+//             fs_utils.create_fresh_path(`${config_root}/${dir}`)));
+// }
+
+/**
+ * write_manual_config_file writes config file directly to the file system without using config FS
```

Comment:
Could you please add in the description in which cases we would use it instead of ConfigFS?
(for example - creating invalid JSON...)